### PR TITLE
Functional tests - Add new test 'CRUD standard product V2'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add new test 'CRUD standard product V2'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| Related PRs       | no
| How to test?      | no need
| Possible impacts? | no

### CODE REVIEW

File: tests/UI/campaigns/productV2/functional/02_CRUSStandardProduct.ts [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+
+// Import common tests
+import loginCommon from '@commonTests/BO/loginBO';
+import {enableNewProductPageTest, disableNewProductPageTest} from '@commonTests/BO/advancedParameters/newFeatures';
+
+// Import faker data
+import ProductFaker from '@data/faker/product';
+import files from '@utils/files';
+
+const baseContext = 'productV2_functional_CRUDStandardProduct';
```
Comment (Progi1984) [inline tests/UI/campaigns/productV2/functional/02_CRUSStandardProduct.ts]: The filename is not correct : `CRUSStandardProduct.ts` => `CRUDStandardProduct.ts`

Comment (nesrineabdmouleh) [reply tests/UI/campaigns/productV2/functional/02_CRUSStandardProduct.ts]: Nice catch thank you Franck :)

File: tests/UI/pages/commonPage.js [hunk 2]
```diff
@@ -255,7 +255,7 @@ class CommonPage {
    * @param browserContext {BrowserContext} Context of the page
    * @param page {Page} Browser tab
    * @param tabId {number} Tab to get focus on after closing the other tab
-   * @return {Promise<void>}
+   * @return {Promise<page>}
```
Comment (Progi1984) [inline tests/UI/pages/commonPage.js]:
```suggestion
   * @return {Promise<Page>}
```

**Review by Progi1984 — COMMENTED** [review]
A small feedback ;)

File: tests/UI/campaigns/productV2/functional/02_CRUSStandardProduct.ts (hunk 1 above)
Comment (Progi1984) [reply tests/UI/campaigns/productV2/functional/02_CRUSStandardProduct.ts]: ;)

File: tests/UI/pages/commonPage.js (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/commonPage.js]: Done :)

**Review by Progi1984 — APPROVED** [review]

**Review by SD1982 — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @nesrineabdmouleh

**Pull request merged**